mono/metadata/metadata-verify.c

```diff
@@ -441,8 +441,6 @@ verify_pe_header (VerifyContext *ctx)
 
 	if (offset > ctx->size - 20)
 		ADD_ERROR (ctx, g_strdup ("File with truncated pe header"));
-	if (read16 (pe_header) != 0x14c)
-		ADD_ERROR (ctx, g_strdup ("Invalid PE header Machine value"));
```

Comment:
How are these errors surfaced? On .NET errors like these are exposed as image load exceptions. I assume Mono doesn't throw?

Reply:
These errors wind up in the editor text log. I do have some stringification that leverages https://github.com/Unity-Technologies/mono/blob/unity-master/mono/metadata/image.c#L2220 but it doesn't really give you much other than "Invalid CIL image"

Reply:
I'll poke around a bit to see if I can bubble this up to the unity side somehow as these messages are a bit more informative.

Reply:
Yeah, I think it would be helpful. Not only it would better point out what goes wrong, it might also lead to a workaround.